Assets/Scripts/Level/Level_Data.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public struct Vector2Int
+{
+    public int x, y;
+    public Vector2Int(int x, int y)
+    {
+        this.x = x;
+        this.y = y;
+    }
+}
+
+[System.Serializable]
+public class Level_Data
+{
+    public string Name;
```

Comment:
```suggestion
    public string Name;
    public int minDimension;
```

Reply:
7 Is hardcoded as the minimal dimension. Sounds sensible, but I suggest a variable.

Reply:
Added the variable